Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [workspace.package]
-version = "5.1.3"
+version = "5.1.4"
```

Comment:
If we are releasing 5.2.0, shouldn't the node version be 5.2.0 as well?

Reply:
If we leave it as is, we won't be able to differentiate the nodes in the [Telemetry dashboard](https://telemetry.polkadot.io/#list/0x81443836a9a24caaa23f1241897d1235717535711d1d3fe24eae4fdc942c092c) and thus, won't be able to confirm what exact node version a validator is running. So, I would update it even if it will take us more time just to be sure we clearly distinguish releases

Reply:
Also, there are notes in the [CHANGELOG.md](https://github.com/Cerebellum-Network/blockchain-node/pull/307/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR22) for 5.2.0 which can make the history unclear (if we leave the current 5.1.4)